link_test.go

```diff
@@ -325,7 +453,6 @@ func TestLinkAddDelGretap(t *testing.T) {
 		LinkAttrs: LinkAttrs{Name: "foo6"},
 		IKey:      0x101,
 		OKey:      0x101,
-		PMtuDisc:  1,
```

Comment:
did you remove this line on purpose? Was it causing a failure?

Reply:
It was on purpose: setting it isn't reflected in the resulting `ip6gretap` device so indeed checking for a match resulted in a failing check. The `IFLA_GRE_PMTUDISC` flag is missing from `net/ipv6/ip_gre.c` in the Linux kernel and I think that is because IPv6 uses different approaches to MTU discovery.